src/components/PurchaseOrder/UpdateOrderErrorModal/UpdateOrderErrorModal.js

```diff
@@ -25,15 +25,23 @@ const UpdateOrderErrorModal = ({ orderNumber, cancel, errorCode }) => {
       footer={footer}
       data-test-update-order-error-modal
     >
-      <FormattedMessage id={`ui-orders.errors.${errorCode}`} />
+      {errors.map(({ code, poLineNumber }) => (
+        <p>
+          <FormattedMessage
+            key={poLineNumber}
+            id={`ui-orders.errors.${code}`}
+            values={{ poLineNumber }}
+          />
+        </p>
+      ))}
     </Modal>
   );
 };
 
 UpdateOrderErrorModal.propTypes = {
   orderNumber: PropTypes.string.isRequired,
   cancel: PropTypes.func.isRequired,
-  errorCode: PropTypes.string.isRequired,
+  errors: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
Some misleading logic here, prop is required, but you set a default value above `errors = []`

Reply:
updated